plugins/generator-1.17.1/forge-1.17.1/templates/item/item.java.ftl

```diff
@@ -52,10 +52,25 @@ import net.minecraft.world.entity.ai.attributes.Attributes;
 				.fireResistant()
 				</#if>
 				.rarity(Rarity.${data.rarity})
+				<#if data.isFood>
+				    .food((new FoodProperties.Builder())
+				        .nutrition(${data.nutritionalValue})
+				        .saturationMod(${data.saturation}f)
+                        <#if data.isAlwaysEdible>.alwaysEat()</#if>
+                    	<#if data.forDogs>.meat()</#if>
+                    	.build())

```

Comment:
Consider improving those indentations (same for [1.16.5](https://github.com/MCreator/MCreator/pull/2002/files#diff-de4152b14b40abebf0564d8e7738ae71ce523051c716456217eeca4a431785e2R114-R117)):
```suggestion
				.food((new FoodProperties.Builder())
						.nutrition(${data.nutritionalValue})
						.saturationMod(${data.saturation}f)
						<#if data.isAlwaysEdible>.alwaysEat()</#if>
						<#if data.forDogs>.meat()</#if>
						.build())
```